### ISSUE TITLE
reverseGeocode's NativeGeocoderReverseResult is incomplete

### ISSUE THREAD

**Issue by JuanFLZ** [sebastianbaar/cordova-plugin-nativegeocoder#34]
The response to the reverseGeocode function is incomplete and does not return the fields _thoroughfare_ and _subThoroughfare_

This is my code:
`this.nativeGeocoder.reverseGeocode(position.coords.latitude, position.coords.longitude, { useLocale: true, maxResults: 1 })
          .then((result: NativeGeocoderReverseResult[]) => {
            console.log(JSON.stringify(result));
          })
          .catch((error: any) => console.log(error));`

the previous code only returns:
`[{"countryCode":"ES","countryName":"España","postalCode":"08013","locality":"Barcelona","subLocality":"Barcelona"}]`

**Comment by sebastianbaar** [comment]
Hey @GabberMan,
thanks for creating the issue. Yes, it's a bug on Android. If any of the fields (e.g. thoroughfare) is null, it is not added to the return object. I will fix this and return an empty String - like I do on iOS. I will try to fix this as soon as possible.
Cheers

**Comment by sebastianbaar** [comment]
Fixed in [v3.1.3](https://github.com/sebastianbaar/cordova-plugin-nativegeocoder/releases/tag/v3.1.3)

**Comment by surajsingh23** [comment]
The response to the reverseGeocode does not return the fields thoroughfare and subThoroughfare.
this is my code.
```
reverseGeoCoding(lat, lng) {
    this.nativeGeocoder.reverseGeocode(lat, lng).then(resp => {
      this.name = resp[0].thoroughfare + " ," + resp[0].locality + " ," + resp[0].subLocality + " ," + resp[0].administrativeArea + " ," + resp[0].countryName;
    })
  }
```
